packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1448,20 +1448,42 @@ const consistentPatchProperties = (patchOp, _opts, ctx) => {
 
 const SYNC_DELETE_RESPONSES = ["200", "204", "default"];
 const LR_DELETE_RESPONSES = ["202", "204", "default"];
-const SYNC_ERROR = "Synchronous delete operations must have responses with 200, 204, and default return codes. They also must have no other response codes.";
-const LR_ERROR = "Long-running (LRO) delete operations must have responses with 202, 204, and default return codes. They also must have no other response codes.";
+const SYNC_ERROR = "Synchronous delete operations must have responses with 200, 204 and default return codes. They also must have no other response codes.";
```

Comment:
Curious - What are the "default return codes"?

Reply:
This is referring to the default response. See the Default Response section on this page: https://swagger.io/docs/specification/describing-responses/